spaces/S000205/README.md

```diff
@@ -0,0 +1,8 @@
+---
+uid: S000205
+name: Warsaw circle
+refs:
+  - wikipedia: Shape_theory_(mathematics)#Warsaw_circle
+    name: Warsaw circle on Wikipedia
+---
+The Extended Topologist's Sine Curve is the set $\{(x, \sin\frac{1}{x})\ |\ x \in (0,1]\} \cup \{(0,y)\ |\ y \in [-1,2]\} \cup \{(x,2)\ |\ x\in[0,1]\} \cup \{(1,x) \|\ x \in [\sin(1),2]\}$ with the subspace topology inherited from $\mathbb{R}^2$.
```

Comment:
```suggestion
The set $\{(x, \sin\frac{1}{x})\ |\ x \in (0,1]\} \cup \{(0,y)\ |\ y \in [-1,2]\} \cup \{(x,2)\ |\ x\in[0,1]\} \cup \{(1,x) \|\ x \in [\sin(1),2]\}$ with the subspace topology inherited from $\mathbb{R}^2$.
```

Reply:
The definition is similar to [S114](https://topology.pi-base.org/spaces/S000114) so I would keep the definition similar.
(I forgot to swap "Extended Topologist's Sine Curve" with "Warsaw Circle" when copying